packages/docs/components/link.md

```diff
@@ -1,96 +1,202 @@
-::: slot nimatron-all
-
-# Link
-
-Links are navigation elements displayed as text. A link can can be used to bring a user to another page or initiate a download.
-
-<figure class="nimatron--example">
-  <div class="nimatron--rendered">
-    <p>If you need more info, <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link">view the spec on MDN</a>.</p>
-  </div>
-
-  ```html
-  If you need more info, <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link">view the spec on MDN</a>.
-  ```
-
-</figure>
+---
+template: component
+id: component-link
+title: Link
+description: Links are navigation elements displayed as text. A link can open another page or jump to a section of a page.
```

Comment:
Which of these ends up where?

Reply:
description updates the meta tag, as well as the card description, lead is used as introductory copy in the page header (under the headline)

Reply:
Should we doc all of this somewhere? Would definitely help with authorship and theme maintenance

Reply:
We should, for sure. I am planning to write a README for the theme package.